corehq/apps/reports/static/reports/js/bootstrap5/base.js

```diff
@@ -33,9 +35,11 @@ $(function () {
         reportConfigsView.setConfigBeingViewed(reportConfigModels.reportConfig(defaultConfig));
     }
 
-    $('#email-enabled').tooltip({  /* todo B5: plugin:tooltip */
-        placement: 'right',
-        html: true,
-        title: gettext("You can email a saved version<br />of this report."),
-    });
+    if ($('#email-enabled').length) {
```

Comment:
Nitpick: this could be `document.getElementById`, it's an easy place to drop jQuery.

Reply:
updated